static/js/zamboni/devhub.js

```diff
@@ -1479,3 +1480,77 @@ function initInAppConfig($dom) {
                 dataType: 'text'});
     }));
 }
+
+function initCCLicense() {
+
+    function setCopyright(isCopyr) {
+        // Set the license options based on whether the copyright license is selected.
+        if (isCopyr) {
+            $('.noncc').addClass('disabled');
+            // Choose "No" and "No" for the "commercial" and "derivative" questions.
+            $('input[name="cc-noncom"][value=1], input[name="cc-noderiv"][value=2]').prop('checked', true);
+        } else {
+            $('.noncc').removeClass('disabled');
+        }
+    }
+    function setLicenseFromWizard() {
+        cc_data = $('input[name^="cc-"]:checked').map(function() {
+            return this.dataset.cc}).get();
+        radio = $('#submit-describe #license-list input[type=radio][data-cc="'+cc_data.join(' ')+'"]');
```

Comment:
Spaces around `+` here and in the rest of the file

Reply:
https://github.com/mozilla/addons-server/pull/6740/commits/c1995a248a6ef7e288f102b2debd7e0f4dcbd12c